<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>641</width>
    <height>493</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>4</y>
      <width>66</width>
      <height>16</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>重选串口需要重新打开</string>
    </property>
    <property name="text">
     <string>本地主机IP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnOpen">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>129</y>
      <width>103</width>
      <height>39</height>
     </rect>
    </property>
    <property name="text">
     <string>监听端口</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>44</y>
      <width>66</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>端口</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbPortName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>22</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>391</y>
      <width>48</width>
      <height>15</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>更改周期后需要重新发送</string>
    </property>
    <property name="text">
     <string>周期(ms)</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ckTimestamp">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>105</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>时间戳</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtInterval">
    <property name="geometry">
     <rect>
      <x>64</x>
      <y>389</y>
      <width>41</width>
      <height>19</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="ckSendTimer">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>371</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="text">
     <string>定时发送</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ckRecvHex">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>87</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>十六进制显示</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClearRecv">
    <property name="geometry">
     <rect>
      <x>477</x>
      <y>295</y>
      <width>71</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>清空接收</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClearSend">
    <property name="geometry">
     <rect>
      <x>557</x>
      <y>294</y>
      <width>72</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>清空发送</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chSendHex">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>331</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>十六进制发送</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chSendNewline">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>351</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>发送回车换行</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txtSend">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>330</y>
      <width>515</width>
      <height>138</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txtRecv">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>5</y>
      <width>514</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cbPortName_2">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>62</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>204</y>
      <width>66</width>
      <height>16</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>重选串口需要重新打开</string>
    </property>
    <property name="text">
     <string>远程主机IP</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>245</y>
      <width>66</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>端口</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbPortName_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>223</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="cbPortName_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>263</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSend">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>426</y>
      <width>103</width>
      <height>39</height>
     </rect>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnConnect">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>289</y>
      <width>103</width>
      <height>39</height>
     </rect>
    </property>
    <property name="text">
     <string>连接</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>641</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
